{"name": "test_create_task_without_name", "status": "passed", "description": "Проверка создания задачи без имени — должен вернуть ошибку", "steps": [{"name": "Создаем задачу без имени", "status": "passed", "start": 1749921073881, "stop": 1749921074013}], "start": 1749921073881, "stop": 1749921074013, "uuid": "8e3490b8-bd59-4857-bf54-47ba8e932f3d", "historyId": "19c5729ffa9951a80e22d57809024d95", "testCaseId": "19c5729ffa9951a80e22d57809024d95", "fullName": "tests.api.test_tasks.TestTasks#test_create_task_without_name", "labels": [{"name": "feature", "value": "Clickup API: Tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "A1"}, {"name": "thread", "value": "8288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}